app/components/admin/users/tools/credits_component.html.erb

```diff
@@ -0,0 +1,144 @@
+<section
+  class="crayons-card p-3 js-component"
+  data-controller="users--tools--ajax"
+  data-action="ajax:success@document->users--tools--ajax#success ajax:error@document->users--tools--ajax#error">
+  <header>
+    <h3><%= link_to "← Credits", admin_user_tools_path(@user),
+                    remote: true,
+                    data: { action: "ajax:success@document->user#replacePartial" },
+                    class: "crayons-title fs-l" %></h3>
+  </header>
+
+  <%# Add user credits %>
+  <%= form_with url: admin_user_tools_credits_path(@user),
+                method: :post,
+                remote: true,
+                class: "my-3",
+                id: :credits_add do |f| %>
+    <fieldset>
+      <legend class="fs-base fw-medium mb-3"><%= pluralize(@user_unspent_credits_count, "available user credit") %></legend>
```

Comment:
I don't think this should be the `legend` as it's more supplemental info than describing the input group

Reply:
A generic div then?